doc-content/shared-kie-docs/src/main/asciidoc/KieServer/controller-java-api-requests-proc.adoc

```diff
@@ -0,0 +1,120 @@
+[id='controller-java-api-requests-proc_{context}']
+= Sending requests with the {CONTROLLER} Java client API
+
+The {CONTROLLER} Java client API enables you to send REST or WebSocket requests to the {CONTROLLER} from your Java client application. You can use the {CONTROLLER} Java client API as an alternative to the {CONTROLLER} REST API to interact with your {KIE_SERVER} templates (configurations), {KIE_SERVER} instances (remote servers), and associated KIE containers (deployment units) in {PRODUCT} without using {CENTRAL}.
```

Comment:
perhaps: enables you to connect to the  {CONTROLLER} using REST or WebSocket protocols from...

Reply:
Done. Updated similarly in KIE Server Java API section.